incskill_site/views.py

```diff
@@ -112,7 +112,8 @@ def post(self, request):
 class ProfileView(View):
     template_name = 'profile.html'
     def get(self, request):
-        if request.user.is_authenticated:
+        if request.user.is_authenticated:            
```

Comment:
```suggestion
        if request.user.is_authenticated:
```
thjere are a bunch of random spaces at the end of this line